<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="rad1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>105</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rad2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>105</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radA">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>154</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radB">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>154</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
